### DISCUSSION TITLE
Passing the props.name to Select?

### DISCUSSION THREAD

**Discussion by dshmatok** [mantinedev/mantine#1767] [Q&A]
Hello! 
First of all, thanks for the wonderful library of components! 🥇 

I have a question about using `props.name` in `Select` component. 
![image](https://user-images.githubusercontent.com/60958933/178266953-0e462043-e6d1-4e1a-97ee-d2065889d789.png)
It is not in the description, but the transmitted value is assigned to the hidden element.
https://github.com/mantinedev/mantine/blob/aac760de2678e749c3655614e1e24b288856770b/src/mantine-core/src/components/Select/Select.tsx#L582

It would be nice if the transmitted `props.name` was assigned to the main element and it could be worked with in the `onBlur` handler. Currently the value is missing: 
https://github.com/mantinedev/mantine/blob/aac760de2678e749c3655614e1e24b288856770b/src/mantine-core/src/components/Select/Select.tsx#L607
Is it possible to add a value transfer to the main element? Thanks!

**Comment by rtivital** [comment]
No, main element is used to display the label of select item, not the actual value of Select.

**Reply by dshmatok** [reply]
I understand. Thanks!

But how I'm can get a `onBlur` event so that `event.target.name` has a passed value from `props.name`?
